frontend/src/components/ingress/Details.tsx

```diff
@@ -74,7 +98,7 @@ export default function IngressDetails() {
               columns={[
                 {
                   label: t('Host'),
-                  getter: (data: IngressRule) => data.host || '*',
+                  getter: (data: IngressRule) => isFormatURL(data.host || '*'),
```

Comment:
Should we include the path too? There's something to be decided about that here.

Reply:
Should we add the same path as the field below this?

Reply:
I think you'd almost always want the path in the link. It doesn't seem to me very common that someone would want to visit just the host part.

Reply:
There is a link for the host field and for the path field!